core/pxt_blockly_functions.js

```diff
@@ -395,6 +395,19 @@ Blockly.Functions.makeEditOption = function(block) {
   return editOption;
 };
 
+Blockly.Functions.makeGoToDefinitionOption = function(block) {
+  var gtdOption = {
+    enabled: true,
+    text: Blockly.Msg.FUNCTIONS_GO_TO_DEFINITION_OPTION,
+    callback: function() {
+      var functionName = block.getField("function_name").getText();
```

Comment:
should this be getValue?

Reply:
should it? I just copied what we do in blocklycompiler

Reply:
HMM no i think you're right, gettext is safer for compatibility with old functions